### PR TITLE
admin dev readme

### PR DESCRIPTION
**Pull request by jspaaks**
Refs #349

### CODE REVIEW

**Comment by jspaaks** [comment]
Doing some more work on this, converted back to draft

**Comment by jspaaks** [comment]
Hi @sverhoeven I feel this PR is a step in the right direction but it's not there yet. Could you give me some pointers?

File: admin/README.md [hunk 1]
```diff
@@ -1,61 +1,86 @@
...
 
-Run-time settings should be reachable through `[BASE_URL]/settings.json`, see e.g. `public/settings.json`.
-This settings file should be overriden in production.
+1.  `cd admin`
+1.  Install yarn, `npm install -g yarn` (Note: there exists a similarly named package, Apache Hadoop YARN --please ignore
+    that, it's something else)
+1.  Install the `admin` service's dependencies: `yarn install`
+1.  In `package.json`, replace (only) the `localhost` part of the URLs listed under `proxy` with the IP addresses for
+    `/api` using `backend`'s IP and for `auth` using `auth`'s IP; leave the port numbers as they were
+1.  Start the admin service in a development server: `yarn start`. It will tell you where to go to check the `admin`
```
Comment (sverhoeven) [inline admin/README.md:62]: To login in to admin interface you need to set the callback to `http://localhost:8000/auth/get_jwt` on https://github.com/settings/developers

File: admin/README.md [hunk 2]
```diff
@@ -1,61 +1,86 @@
...
-## Settings
+    # where is backend running?
+    docker inspect $(docker-compose ps -q backend) | \
+       jq .[0].NetworkSettings.Networks.rsd_default.IPAddress
+    ```
 
-Run-time settings should be reachable through `[BASE_URL]/settings.json`, see e.g. `public/settings.json`.
-This settings file should be overriden in production.
+1.  `cd admin`
+1.  Install yarn, `npm install -g yarn` (Note: there exists a similarly named package, Apache Hadoop YARN --please ignore
```
Comment (sverhoeven) [inline admin/README.md:57]: This is more a prereq, please move to prerequisite chapter.

**Review by sverhoeven — CHANGES_REQUESTED** [review]
I was hoping for something simpler. 

For example using docker-compose to start the required services on local ports defined in package.json:proxy. See https://github.com/research-software-directory/research-software-directory/pull/610 for simpler to dev experience, though slightly more complex setup.

**Comment by jspaaks** [comment]
Hi @sverhoeven, thanks for the review. I still have to try out #610, but from scrolling through the diff it seems that is a better way to go than this PR. Shall I just mark this one as closed?

**Comment by sverhoeven** [comment]
Let's close this one. And I will directly merge https://github.com/research-software-directory/research-software-directory/pull/610 to master.

**Comment by jspaaks** [comment]
:+1:

**Pull request closed without merging**